packages/grafana-data/src/valueFormats/valueFormats.ts

```diff
@@ -156,6 +156,10 @@ export function simpleCountUnit(symbol: string): ValueFormatter {
   };
 }
 
+export function stringFormater(value: number): FormattedValue {
+  return { text: value as any };
```

Comment:
or maybe expicitly a string?
```suggestion
  return { text: `${value}` };
```

or:
```suggestion
  return { text: ''+value };
```